src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -179,3 +181,74 @@ describe('test conversion of DatasetAccessRequest', () => {
     expect(convertDatasetAccessRequest(datasetAccessRequest)).toStrictEqual(datasetAccessRequestApi);
   });
 });
+
+describe('test HighlightConceptName', () => {
```

Comment:
I also saw tests for HighlightConceptName in the ConceptSearch tests. Do we need both of them?

Reply:
Maybe I can remove all but a test for bolding from the ConceptSearch test code, there are overlaps and the important thing for that page is that the component is set up correctly, rather than needing to test all cases there.